src/apps/xpubs/xpubs.py

```diff
@@ -93,6 +94,15 @@ async def menu(self, show_screen, show_all=False):
                           "Public keys are saved to the file:\n\n%s" % filename,
                           button_text="Close")
                 )
+	elif menuitem == 4:
+	    from_account = await show_screen(NumericScreen(title="Enter FROM account number", current_val=str(self.account)))
+	    to_account = await show_screen(NumericScreen(title="Enter TO account number", current_val=str(self.account)))
+	    if from_account is None or to_account is None:
+	        return 
+	    from_account = int(from_account)
```

Comment:
Handle `from_account=""` correctly (if I just accepted the default):

```suggestion
	    from_account = int(from_account or self.account)
```

Reply:
Oh, by the way, makes sense to do the same for "to_account" as well.